keras_nlp/models/whisper/whisper_multi_head_attention.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2023 The KerasNLP Authors
```

Comment:
We should be able to remove this https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/whisper/whisper_backbone_test.py#L78 as part of this PR